source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -33,6 +41,11 @@ In general, the XML file must meet the following requirements:
 <a href="#xml-example">Follow this link</a> for a valid XML example or
 <a href="#xsd">read</a> the XSD definition of the XML.
 
+<% if settings.feed_validator_domain %>
+Alternatively, <%= link_to "follow this link", "http://#{settings.feed_validator_domain}" %>
+to validate your XML feed against <%= settings.site_name.capitalize %> requirements.
```

Comment:
Fixed. I'm using "standards" as this is how it is referred elsewhere in the document. Specification mostly refers to the XML language specification and/or the Skroutz feedspec as a whole.